### ISSUE TITLE
js-toolkit/docs/manuals/liferay-cli.md doesn't specify node version

### ISSUE THREAD

**Issue by pat270** [liferay/liferay-frontend-projects#922]
> 👀 Please check the [troubleshooting guide](https://github.com/liferay/liferay-frontend-projects/blob/master/maintenance/projects/js-toolkit/docs/How-to-troubleshoot-your-setups.md) before reporting anything. It contains important information on how to fix or diagnose errors.

### Issue type (mark with `x`)

-   [ ] :thinking: Question
-   [ ] :bug: Bug report
-   [ ] :gift: Feature request
-   [x ] :woman_shrugging: Other

### Version (mark with `x`)

-   [ ] :two: v2.x
-   [ ] :three: v3.x

### Description
A person in the community Slack tried to start a new React Project with node 16.4.2 and ran into errors.
**Desired behavior:**
Document the range of node versions `@liferay/cli` is compatible with.
**Current behavior:**
There is no documentation.

**Comment by WHI-CH** [comment]
I had the issue.
Trying to build a project with Node 16.x returns an error, but switching to Node 14.19.1 with nvm works.
If current versions aren't supported this should be documented (but ideally, they should be supported).

**Comment by WHI-CH** [comment]
I'm having the problem, added [a new issue here](https://github.com/liferay/liferay-frontend-projects/issues/923)

**Comment by izaera** [comment]
I don't think this is a node version problem, because I've been using node 16 for quite a long time and never experienced an issue.

We officially support Node LTS (which currently is 16) so this is not a matter of documentation but of finding the solution to #923.

In any case, I'll leave this open to add the LTS thingie to the docs :+1:
